_docs/set-up-file-provisioning-plugins.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Set Up File Provisioning Plugins
+permalink: /docs/publisher-tutorials/set-up-file-provisioning-plugins/
+---
+
+# Set Up File Provisioning Plugins
+
+File provisioning plugins can be used to extend the Dockstore CLI to allow for input and output files to be downloaded or uploaded to new and/or proprietary services. 
+By default, the Dockstore CLI tries to read input/output files from the local filesystem and copy/hard-link to them. If that does not work, it will look at whether any of the installed
+plugins can handle the files. Finally, it will try using [commons-vfs2](https://commons.apache.org/proper/commons-vfs/filesystems.html) to handle the files.
+
+Generally, the Dockstore CLI downloads a default set of file provisioning plugins, Amazon S3, Synapse, DOS, GS, and ICGC Storage Client, when it first installs.
+However, if you chose to download and configure the CLI yourself via step 3 of Part 1 during the [quickstart](https://dockstore.org/quick-start) tutorial and did not have a Dockstore account set up,
+then no file provisioning plugins will have downloaded. To check if you have the plugins, run the command `dockstore plugin list`. You should see something like
+
+![file provisioning plugins](/assets/images/docs/file-provisioning-plugins.png)
```

Comment:
You optionally could just use the triple ticks to display console output instead of having an image here.

Reply:
The table gets squished when I used the triple ticks. I wasn't sure how to format it so it'd look okay. Ended up just taking a screenshot.